lahja/trio/endpoint.py

```diff
@@ -599,12 +597,12 @@ def is_server_stopped(self) -> bool:
         del self._server_nursery
 
         try:
-            await self.ipc_path.unlink()
+            self.ipc_path.unlink()
```

Comment:
Alternatively, we could continue using the async API (via trio.Path) but change the try/except to catch trio.Cancelled, then synchronously unlink the file and re-raise trio.Cancelled. But I really don't see the point of using the async API here in the first place

Reply:
I'm not familar with the trio APIs tbh. From what I'm seeing here I agree that synchronously unlinking should be fine. But maybe @pipermerriam who wrote the trio implementation has more insight.

Reply:
I don't think the async filesystem APIs are actually that good in trio right now so I think this is just fine.  Last I remember the main trio maintainer said they were implemented using threads and were kind of gross.  :+1: